cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -162,6 +163,25 @@ instance DefineSeverity (WithDomainName DnsTrace) where
     DnsTraceLookupAResult _    -> Debug
     DnsTraceLookupAAAAResult _ -> Debug
 
+instance DefinePrivacyAnnotation (WithMuxBearer (MuxTrace ptcl))
+instance DefineSeverity (WithMuxBearer (MuxTrace ptcl)) where
+  defineSeverity (WithMuxBearer _ ev) = case ev of
+    MuxTraceRecvHeaderStart      -> Debug
+    MuxTraceRecvHeaderEnd _      -> Debug
+    MuxTraceRecvPayloadStart _   -> Debug
+    MuxTraceRecvPayloadEnd _     -> Debug
+    MuxTraceRecvStart _          -> Debug
+    MuxTraceRecvEnd _            -> Debug
+    MuxTraceSendStart _          -> Debug
+    MuxTraceSendEnd              -> Debug
+    MuxTraceStateChange _ _      -> Info
+    MuxTraceCleanExit _          -> Info
+    MuxTraceExceptionExit _ _    -> Info
```

Comment:
this could be `Error` ?

Reply:
No, it is not an error for a bearer to exit due to an exception (examples of such exceptions are when the peer closes the connection or chainsync thinks that the peer is too out-of-sync for us to be talking to it).